scanner/k8s-assets/scanner/config.go

```diff
@@ -0,0 +1,10 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package scanner
+
+type Config struct {
+	KubeConfigPath    string `envconfig:"KUBE_CONFIG_PATH" default:"~/.kube/config" required:"true" json:"-"`
```

Comment:
Can we load this one in the OIDC config and add another config to say whatever the OIDC config should be used or not?